node/full.go

```diff
@@ -146,6 +149,16 @@ func (qm *QitmeerFull) RegisterAccountService(cfg *config.Config) error {
 	return nil
 }
 
+func (qm *QitmeerFull) RegisterWalletService(cfg *config.Config, conf node.Config, _am *acct.AccountManager, _tm *tx.TxManager) error {
```

Comment:
should not export the `github.com/ethereum/go-ethereum/node` here, from the `node.Config`
you might have to refactor the `wallet.New` API. and using the `go-ethereum` internally.